spec/lib/mastodon/cli/maintenance_spec.rb

```diff
@@ -52,5 +52,78 @@
           .and raise_error(SystemExit)
       end
     end
+
+    context 'when requirements are met' do
+      before do
+        allow(ActiveRecord::Migrator).to receive(:current_version).and_return(2023_08_22_081029) # The latest migration before the cutoff
```

Comment:
Should we really do that? It means it may run on a schema that is not what it was written for.